frontend/src/pages/BatchSend.vue

```diff
@@ -0,0 +1,538 @@
+<template>
+  <q-page v-if="isMaintenanceMode" padding>
+    <div
+      class="dark-toggle form-max-wide text-center text-bold q-pa-md"
+      style="border-radius: 15px"
+      :style="isDark ? 'color: #FFEEEE; background-color: #780A0A' : 'color: #610404; background-color: #FACDCD'"
+    >
+      {{ $t('Send.sending-disabled') }}
+    </div>
+  </q-page>
+
+  <q-page v-else padding>
+    <h2 class="page-title">Batch Send</h2>
```

Comment:
We'll need translations for all of these, do you want to do that in this PR or track separately?

Reply:
I'll do it in this PR, but haven't addressed it yet. Leaving this open for now

Reply:
Fixed them here: https://github.com/ScopeLift/umbra-protocol/pull/491/commits/bb9b4b6f027fb57c4008bda8d1d2783d0d87029a

Reply:
I've noticed you marked this ready for re-review last week so just bumping this as a reminder

Reply:
Added here: https://github.com/ScopeLift/umbra-protocol/pull/491/commits/136d2be8b555bb1c501646e75e6c2f1494a9261e